app/grandchallenge/evaluation/forms.py

```diff
@@ -431,14 +431,26 @@ class Meta:
 
 
 class LegacySubmissionForm(SubmissionForm):
-    def __init__(self, *args, challenge, **kwargs):
+    def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
         self.fields[
             "creator"
-        ].queryset = challenge.participants_group.user_set.all().order_by(
+        ].queryset = self._phase.challenge.participants_group.user_set.all().order_by(
             Lower("username")
         )
 
+    def clean(self):
+        if Evaluation.objects.filter(
+            submission__phase__challenge=self._phase.challenge,
+            status=Evaluation.EXECUTING_PREREQUISITES,
```

Comment:
Why only this status, and not also `PENDING`, `PROVISIONING` and `EXECUTING`?

Reply:
They don't cause problematic load on the system - it's one evaluation to hundreds of algorithm jobs, and it's the latter we want to stop.

Reply:
Ah of course, that makes sense! Thanks for explaining!